src/main/java/com/authservice/config/JwtEncryption.java

```diff
@@ -0,0 +1,19 @@
+package com.authservice.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+
+@Component
+public class JwtEncryption {
+
+    @Value("${app.jwt.secret}")
+    private String jwtSecret;
+
+    public String encodeSecretKey() {
```

Comment:
getEncodeSecretKey()

Reply:
Updated changes: Completed  and rename  method name to getEncodeSecretKey()